public/assets/css/show.css

```diff
@@ -0,0 +1,115 @@
+* {
+    box-sizing: border-box;
+    margin: 0px;
+    padding: 0px;
+}
+
+body {
+    background-color: rgb(232, 232, 232);
+}
+
+#product_page {
+    padding: 70px;
+    display: grid;
+    grid-template: auto/1fr 1fr;
+    column-gap: 70px;
+    font-size: 1.8em;
+}
+
+span {
```

Comment:
please put a class name or put this span in a BEM otherwise all the span of the show page will take this behavior